File: solutions/kotlin/darts/1/src/main/kotlin/Darts.kt
object Darts {

    fun score(x: Number, y: Number): Int {
        val dX = x.toDouble()
        val dY = y.toDouble()
        val dist = Math.sqrt((dX * dX) + (dY * dY))

        return if (dist > 10) {
            0
        } else if(dist > 5) {
            1
        } else if(dist > 1) {
            5
        } else {
            10
        }
    }
}

File: solutions/typescript/darts/1/darts.ts
export function score(x: number, y: number): number {
  const dist = Math.sqrt(x * x + y * y);
  if (dist > 10) {
    return 0;
  } else if (dist > 5) {
    return 1;
  } else if (dist > 1) {
    return 5;
  } else {
    return 10;
  }
}
